src/main/java/com/bootcamp/demo/controller/CardController.java

```diff
@@ -0,0 +1,37 @@
+package com.bootcamp.demo.controller;
+
+import com.bootcamp.demo.model.Card;
+import com.bootcamp.demo.service.CardService;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.cloud.FirestoreClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/card")
+public class CardController {
+    private Firestore firestoreDB;
+    private CardService cardService;
```

Comment:
It's a good ideea to have a service layer that can perform additional operations before calling the persistence layer (DAO / Repository)